Formula/ooniprobe.rb

```diff
@@ -3,287 +3,146 @@ class Ooniprobe < Formula
-    ln_s pkgshare/"decks/web.yaml", pkgshare/"current.deck"
-  end
+      prefix.install_metafiles
+    end
 
-  def caveats
-    <<~EOS
-      Decks are installed to #{opt_pkgshare}/decks.
+    (pkgshare/"etc/ooniprobe-daily-config.json").write <<~EOS
+    {
+      "_version": 3,
+      "_informed_consent": true,
+      "sharing": {
+        "include_ip": false,
+        "include_asn": true,
+        "upload_results": true
+      },
+      "nettests": {
+        "websites_url_limit": 100,
+        "websites_enabled_category_codes": [
+          "ALDR",
+          "ANON",
+          "COMM",
+          "COMT",
+          "CTRL",
+          "CULTR",
+          "DATE",
+          "ECON",
+          "ENV",
+          "FILE",
+          "GAME",
+          "GMB",
+          "GOVT",
+          "GRP",
+          "HACK",
+          "HATE",
+          "HOST",
+          "HUMR",
+          "IGO",
+          "LGBT",
+          "MILX",
+          "MISC",
+          "MMED",
+          "NEWS",
+          "POLR",
+          "PORN",
+          "PROV",
+          "PUBH",
+          "REL",
+          "SRCH",
+          "XED"
+        ]
+      },
+      "advanced": {
+        "use_domain_fronting": false,
+        "send_crash_reports": true,
+        "collect_usage_stats": true,
+        "collector_url": "",
+        "bouncer_url": "https://bouncer.ooni.io"
+      }
+    }
     EOS
   end
 
-  plist_options startup: "true", manual: "ooniprobe -i #{HOMEBREW_PREFIX}/share/ooniprobe/current.deck"
+  plist_options startup: "true", manual: "ooniprobe --config \"#{HOMEBREW_PREFIX}/etc/ooniprobe-daily-config.json\" run"
 
   def plist
     <<~EOS
-      <?xml version="1.0" encoding="UTF-8"?>
-      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-      <plist version="1.0">
-      <dict>
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+    <dict>
         <key>Label</key>
-          <string>#{plist_name}</string>
+        <string>#{plist_name}</string>
+
+        <key>KeepAlive</key>
+        <false/>
+        <key>RunAtLoad</key>
+        <true/>
+
         <key>EnvironmentVariables</key>
         <dict>
           <key>PATH</key>
           <string>#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
         </dict>
+
         <key>ProgramArguments</key>
         <array>
-          <string>#{opt_bin}/ooniprobe-agent</string>
-          <string>run</string>
+            <string>#{opt_bin}/ooniprobe</string>
+            <string>--config "#{HOMEBREW_PREFIX}/etc/ooniprobe-daily-config.json"</string>
+            <string>--batch</string>
+            <string>run</string>
         </array>
-        <key>RunAtLoad</key>
-          <true/>
-        <key>KeepAlive</key>
-          <true/>
+
+        <key>StartInterval</key>
+        <integer>3600</integer>
```

Comment:
Can we make this interval like six hours, at least for the first release?

Reply:
fixed!